ui/pages/asset/asset.tsx

```diff
@@ -29,7 +29,7 @@ const Asset = () => {
 
   const token = findAssetByAddress(selectedAccountTokensChains, asset, chainId);
 
-  const nft = nfts.find(
+  const nft: Nft = nfts.find(
```

Comment:
technically `.find` can also return `undefined`, is this safe?

Reply:
This is a good callout. I think that it's fine for now, since we are conditionally checking for it further down in the file. 

However, I do agree that this is not the best way to check which Asset to render. My next task is to refactor `Asset` and `AssetList` to be chain agnostic. Maybe we can refine this there?

Reply:
Nice, let's handle it in a separate task